runtime/kusama/src/lib.rs

```diff
@@ -1594,7 +1596,7 @@ pub type Executive = frame_executive::Executive<
 	Block,
 	frame_system::ChainContext<Runtime>,
 	Runtime,
-	AllPallets,
+	AllPalletsWithSystem,
```

Comment:
So here we have switched from _reverse_ order, to _not-reversed_ order. I was expecting this to cause some items in construct_runtime to be swapped. How so?

Reply:
the only constraint I have found are those I put into comment, and switching to no-reversed order actually fix them.
I believe than currently the block producer of the last block of an era doesn't get points (except if he is elected in the next era) and also the block producer of last block of a session cannot tell he was online.